profile/README.md

```diff
@@ -0,0 +1,31 @@
+## Hi there 👋
+
+🙋‍♀️ OpenSCD product vision
+
+For utility/integrator/vendor companies​
+
+Who want to have flexibility, agility and efficiency in their SCL configurations​
+
+**OpenSCD**​
+
+is an IEC 61850 SCL configuration tool​
+
+That offers a front-end, open source, extensible, customizable, user friendly, SCL editing​
+
+Unlike proprietary, vendor specific, desktop oriented, limited integration options, not user friendly tools ​
+
+Our product allow offline editing using web-browsers gives flexibility, control over your own roadmap, prevent a vendor-lock-in and uses the power of open source, modern technologies to help the energy transition.
```

Comment:
The first part of the sentence does not fit to the other part, I feel. The technology - web app - we are using itself does not give flexibility, not does it prevent a vendor lock in and so on.

Reply:
Let me try to leave a better, more constructive comment: `Beeing an collborative open-source project the tool gives you flexibility and control over your own raodmap and this ultimatevely prevents any vendor-lock-in. Using the open-source approach to agree on software level will help you to accelerate the energy transition`